src/fides/api/ctl/migrations/versions/c9ee230fa6da_add_config_set_column_to_application_.py

```diff
@@ -0,0 +1,36 @@
+"""add config_set column to application settings
+
+Revision ID: c9ee230fa6da
+Revises: 8e198eb13802
+Create Date: 2023-02-01 15:13:52.133075
+
+"""
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "c9ee230fa6da"
+down_revision = "8e198eb13802"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "applicationconfig",
+        sa.Column(
+            "config_set",
+            sqlalchemy_utils.types.encrypted.encrypted_type.StringEncryptedType(),
+            nullable=False,
+        ),
+    )
```

Comment:
@adamsachs Migration may need to be updated while you're here? - it looks like `api_set` is nullable=True in database, but nullable=False on the model.

Reply:
goot catch! i'd just noticed this myself locally when looking into the weird `ctl-not-external` CI timeouts (unrelated to this i believe). but i'll make the update since yes, i'd missed this in my previous PR for the storage work.

Reply:
ah yeah just noticed it on another branch! thank you!

Reply:
added in https://github.com/ethyca/fides/pull/2551/commits/e890bf28dbc7bc10cdb4cd1708a943edc6c57155